src/core/plugins/icons/components/locked-auth-operation-btn.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * @prettier
+ */
+import React from "react"
+import PropTypes from "prop-types"
+
+const LockedAuthOperationBtn = ({
```

Comment:
Shouldn't this be named `LockedAuthOperationIcon`? This also applies to all other icons

```suggestion
const LockedAuthOperationIcon = ({
```

Reply:
Corrected in latest PR